src/packages/@ncigdc/uikit/Tooltip/GlobalTooltip.js

```diff
@@ -11,12 +12,16 @@ class GlobalTooltip extends Component {
     window.removeEventListener('mousemove', this.moveTooltip);
   }
 
-  moveTooltip = event => {
-    this.globalTooltip.style.left = `${event.pageX}px`;
-    this.globalTooltip.style.top = `${event.pageY -
-      this.globalTooltip.offsetHeight -
-      15}px`;
-  };
+  moveTooltip = _.throttle(event => {
+    const left = event.pageX;
+    const top = event.pageY;
+    setTimeout(() => {
```

Comment:
Does this fix it, or just make the edge case edgier? Would it make sense to add a componentDidUpdate and add/remove the listener based on `props.tooltip.Component`?